tests/test_app_bokeh.py

```diff
@@ -0,0 +1,741 @@
+"""Test the tiatoolbox visualization tool."""
+from __future__ import annotations
+
+import io
+import json
+import multiprocessing
+import re
+import time
+from pathlib import Path
+from typing import TYPE_CHECKING, Generator
+
+import bokeh.models as bkmodels
+import matplotlib.pyplot as plt
+import numpy as np
+import pkg_resources
```

Comment:
```suggestion
import pkg_resources
```

Please replace this as discussed. pkg_resources is being deprecated.

Reply:
i already replaced it where suggested. didn't realise it was present here too